include/superior_mysqlpp/low_level/dbdriver.hpp

```diff
@@ -628,17 +628,6 @@ namespace SuperiorMySqlpp { namespace LowLevel
                 return mysql_field_tell(resultPtr);
             }
 
-            /**
-             * Frees allocated memory for current result set.
-             * @see https://dev.mysql.com/doc/refman/5.7/en/mysql-free-result.html
-             *
-             * @return Void - nothing!
-             */
-            auto freeResult()
```

Comment:
IMHO would be better to leave it here, but call `mysql_free_result` with same condition as in close(), make it noexcept, and possibly remove close() method and instead use this one? @smradCZ ?